packages/flutter_test/lib/src/binding.dart

```diff
@@ -1226,24 +1226,52 @@ class AutomatedTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
 /// frames.
 ///
 /// These values are set on the binding's
-/// [LiveTestWidgetsFlutterBinding.framePolicy] property. The default is
-/// [fadePointers].
+/// [LiveTestWidgetsFlutterBinding.framePolicy] property.
+///
+/// {@template flutter.flutter_test.frame_policy}
+/// The default is [fadePointers]. Setting this to anything other than
+/// [onlyPumps] means pumping extra frames, which might involve calling builders
+/// more, or calling paint callbacks more, etc, which might interfere with the
+/// test. If you know your test file wouldn't be affected by this, you can set
+/// it to [fullyLive] or [benchmarkLive] persistently in that particular test
+/// file.
+///
+/// To set a value while still allowing the test
+/// file to work as a normal test, add the following code to your test file at
```

Comment:
By "a normal test" do you mean one that won't call pump extra times?
Maybe:
```
To set a policy that won't cause the test to pump extra frames,
add the following to the `main()` function, before calling
[testWidgets]: 
```

Reply:
I didn't write this but I think normal here just mean the test is like what we typically write a test, not the kind of test with tricky hack.